### PR TITLE
Add password change form

### PR DESCRIPTION
**Pull request by sarayourfriend**
Necessary because we'll create passwords for people and ask them to change their password. We should look into some way of forcing people to change their password the same way we do for the wiki when we create accounts.

Note: hide whitespace to deal with the formatting changes. The 2 space indentation in the HTML was annoying and not in line with out editorconfig.

### CODE REVIEW

**Comment by sarayourfriend** [comment]
> We should look into some way of forcing people to change their password the same way we do for the wiki when we create accounts.

We can probably do this using some sort of Middleware and adding a boolean to the user `require_password_change` that auto redirects the user to the password change form after they've logged in and haven't yet changed their password.

Something for a different PR though

File: Makefile [hunk 1]
```diff
@@ -96,3 +96,8 @@ install: .env
 .PHONY: lint
 lint:
 	pre-commit run --all-files
+
+
+.PHONY: attach-django
+attach-django:
```
Comment (AetherUnbound) [inline Makefile:102]: What is this used for?

**Review by AetherUnbound — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (sarayourfriend) [reply Makefile:102]: If you add an `import ipdb; ipdb.set_trace()` somewhere you need to attach to the container in order for it to capture your keyboard input (`dc logs` just streams, but doesn't capture). So this makes it easier to attach to the django service.

Comment (AetherUnbound) [reply Makefile:102]: Ahh, got it!

**Pull request merged**